conda_smithy/feedstock_content/.gitignore

```diff
@@ -1,3 +1,13 @@
-*.pyc
+# User content belongs under recipe/.
+# Feedstock configuration goes in `conda-forge.yml`
+# Everything else is managed by the conda-smithy rerender process.
+# Please do not modify
+
+*
+!conda-forge.yml
 
-build_artifacts
+!*/
```

Comment:
What does this line do?

Reply:
It tells git to not ignore subdirectories in the subsequent re-inclusions. Otherwise directories like `recipe/patches/` don't get considered in `!recipe/**` as sub-directory re-inclusions are not considered due to performance reasons.

Reply:
Thanks